cfme/tests/configure/test_analysis_profiles.py

```diff
@@ -24,13 +24,107 @@
     {'Name': files_list[0]['Name'],
      'Collect Contents?': not files_list[0]['Collect Contents?']}]
 
+TENANT_NAME = "tenant_{}".format(fauxfactory.gen_alphanumeric())
+
+PRODUCT_FEATURES = [
+    [
+        (
+            [
+                "Everything",
+                "Settings",
+                "Configuration",
+                "Access Control",
+                "Tenants",
+                "Modify",
+                "Manage Quotas",
+                "Manage Quotas (My Company)",
+            ],
+            False,
+        )
+    ],
+    [
+        (
+            [
+                "Everything",
+                "Settings",
+                "Configuration",
+                "Access Control",
+                "Tenants",
+                "Modify",
+                "Manage Quotas",
+                "Manage Quotas ({})".format(TENANT_NAME),
+            ],
+            False,
+        )
+    ],
+    [
+        (
+            [
+                "Everything",
+                "Automation",
+                "Automate",
+                "Customization",
+                "Dialogs",
+                "Modify",
+                "Add",
+                "Add ({})".format(TENANT_NAME),
+            ],
+            False,
+        )
+    ],
+    [
+        (
+            [
+                "Everything",
+                "Automation",
+                "Automate",
+                "Customization",
+                "Dialogs",
+                "Modify",
+                "Edit",
+                "Edit ({})".format(TENANT_NAME),
+            ],
+            False,
+        )
+    ],
+    [
+        (
+            [
+                "Everything",
+                "Automation",
+                "Automate",
+                "Customization",
+                "Dialogs",
+                "Modify",
+                "Delete",
+                "Delete ({})".format(TENANT_NAME),
+            ],
+            False,
+        )
+    ],
+    [
+        (
+            [
+                "Everything",
+                "Automation",
+                "Automate",
+                "Customization",
+                "Dialogs",
+                "Modify",
+                "Copy",
+                "Copy ({})".format(TENANT_NAME),
+            ],
+            False,
```

Comment:
Why is every one of the second tuple entries `False`? Is it really needed?

Reply:
Yes @mshriver It is required because by default it's true for all and here this test case checks the where unchecking(false) particular tenant feature works or not.

Reply:
Okay, but if its `False` in all cases, why doesn't the test just use `False` directly? Maybe without some kind of comments/labels its just opaque how its being used effectively.

I'm not quite sure what you mean by 'by default its true for all' - default where? Your parametrization of this data structure is local to this module, and there's no default in the data structure, they're just `False`.

Reply:
- Used ```False``` directly in test.
- By default true means: the tree path or nodes in RBAC are already checked (true) to the elements on which i am performing this test.